dev-tools/package.json

```diff
@@ -1,18 +1,22 @@
 {
   "scripts": {
-    "watch:static": "yarn watch \"./bw-dev collectstatic\" bookwyrm/static/**"
+    "css-build": "node-sass --omit-source-map-url ../bookwyrm/sass/bookwyrm.scss ../bookwyrm/static/css/bookwyrm.min.css --output-style=compressed",
+    "css-watch": "npm run css-build -- --watch",
+    "watch:static": "watch \"cd .. && ./bw-dev collectstatic\" ../bookwyrm/static/**"
   },
   "devDependencies": {
+    "bulma": "^0.9.3",
     "eslint": "^8.9.0",
+    "merge": "2.1.1",
+    "node-sass": "^7.0.1",
```

Comment:
We could perhaps try with `sass` instead of the deprecated `node-sass`, what do you think?

https://www.npmjs.com/package/sass